src/bh/components/BhCard.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <div class="bh">
+    <div class="bh-card">
+      <div class="bh-card__left">
```

Comment:
Create ```preprend``` and ```append``` slots.
Prepend and append slots shoud be blocks with minimum width and optionals.
Default slot shoud be required.
Align slots in top by default, but add prop responsible per customization.